app/__init__.py

```diff
@@ -1,17 +1,29 @@
-from flask import Flask, render_template
-from flask_sqlalchemy import SQLAlchemy
 from config import config
+from flask import Flask
+from flask_bootstrap import Bootstrap
+from flask_sqlalchemy import SQLAlchemy
 from flask_login import LoginManager
```

Comment:
According to [PEP8 docs](https://www.python.org/dev/peps/pep-0008/#imports), 
Imports should be grouped in the following order:

* standard library imports
* related third party imports
* local application/library specific imports
* You should put a blank line between each group of imports.

In your case, `from config import Config` should be imported last after a blank line from the flask imports.